internal/configs/virtualserver_test.go

```diff
@@ -2412,3 +2412,33 @@ func TestGenerateQueueForPlus(t *testing.T) {
 	}
 
 }
+
+func TestGenerateSessionCookie(t *testing.T) {
+	tests := []struct {
+		sc   *conf_v1alpha1.SessionCookie
+		want *version2.SessionCookie
```

Comment:
the other tests use expected/result instead want/got convention. Please stick with the former.